OpenRA.Mods.Common/Traits/World/ColorPickerManager.cs

```diff
@@ -61,15 +64,36 @@ public void RulesetLoaded(Ruleset rules, ActorInfo ai)
 				throw new YamlException("PresetHues and PresetSaturations must have the same number of elements.");
 		}
 
-		public IEnumerable<Color> PresetColors()
+		IEnumerable<Color> IColorPickerManagerInfo.PresetColors()
 		{
 			for (var i = 0; i < PresetHues.Length; i++)
-				yield return Color.FromAhsv(PresetHues[i], PresetSaturations[i], V);
+				yield return Color.FromAhsv(PresetHues[i], PresetSaturations[i], PresetValues[i]);
 		}
 
-		public Color Color;
+		Color IColorPickerManagerInfo.ColorPickerPaletteColor { get; set; }
```

Comment:
Why was this renamed? IMO `ColorPickerPaletteColor` is just adding unnecessary redundancy, (`colorManager.ColorPickerPaletteColor` vs `colorManager.Color`).

Reply:
This color is just used in for the color picker palette. An indirect way of passing a value. I don't like it, but IMO at the minimum the name should reflect it